src/components/StyledHeader.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const StyledHeader = styled.div`
```

Comment:
If this is called `StyledHeader` I suggest to use a `header` element instead of a `div` 🙂